tests/conftest.py

```diff
@@ -68,6 +68,7 @@ def pytest_configure(config):
         PASSWORD_HASHERS=(
             'django.contrib.auth.hashers.MD5PasswordHasher',
         ),
+        **use_l10n,
```

Comment:
Perhaps we should just keep this anyways, just to keep it simple?

```suggestion
        USE_L10N=True,  # This is now the default from Django 4.0 onwards.
```

Reply:
It'll raise a warning since `USE_L10N` is being removed in Django 5.0. (And then we have to `ignore_warnings` and ...)

Reply:
Ah gotcha, I guess that one's probably worth a comment.

Reply:
Comment added in 4916854492e6c999977b8577b5a15e6ffc784550